### PR TITLE
chore: prep release v0.25

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.25.0] - 2022-11-16
+
+This release resolves the `parity-util-mem crate` several version guard by updating substrate related dependencies which makes
+it possible to have other substrate dependencies in tree again.
```
Comment (niklasad1) [inline CHANGELOG.md:10]:
```suggestion
it possible to have other substrate dependencies in tree again along with subxt.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ All notable changes to this project will be documented in this file.
...
+- Upgrade to scale-info 2.3 and fix errors ([#704](https://github.com/paritytech/subxt/pull/704))
+- No need to entangle Signer and nonce now ([#702](https://github.com/paritytech/subxt/pull/702))
+- error: `RpcError` with custom client error ([#694](https://github.com/paritytech/subxt/pull/694))
+- into_encoded() for consistency ([#685](https://github.com/paritytech/subxt/pull/685))
+- make subxt::Config::Extrinsic Send ([#681](https://github.com/paritytech/subxt/pull/681))
+- Refactor CLI tool to give room for growth ([#667](https://github.com/paritytech/subxt/pull/667))
+- expose jsonrpc-core client ([#672](https://github.com/paritytech/subxt/pull/672))
+- Upgrade clap to v4 ([#678](https://github.com/paritytech/subxt/pull/678))
+
+
```
Comment (niklasad1) [inline CHANGELOG.md:40]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,39 @@ All notable changes to this project will be documented in this file.
...
+it possible to have other substrate dependencies in tree again along with subxt.
+
+In addition the release has several API improvements in the dynamic transaction API along with that subxt now compiles down to WASM.
+
+Notable PRs merged:
+
+### Added
+
+- Add getters for `Module` ([#697](https://github.com/paritytech/subxt/pull/697))
+- add wasm support (#700)  ([#700](https://github.com/paritytech/subxt/pull/700))
```
Comment (niklasad1) [inline CHANGELOG.md:19]:
```suggestion
- add wasm support ([#700](https://github.com/paritytech/subxt/pull/700))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.25.0] - 2022-11-16
+
+This release resolves the `parity-util-mem crate` several version guard by updating substrate related dependencies which makes
+it possible to have other substrate dependencies in tree again along with subxt.
+
+In addition the release has several API improvements in the dynamic transaction API along with that subxt now compiles down to WASM.
```
Comment (ascjones) [inline CHANGELOG.md:12]: Love the casual mention of the most important feature of all: Wasm compilation :heart:

**Review by ascjones — APPROVED** [review]
LGTM

File: RELEASING.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ We also assume that ongoing work done is being merged directly to the `master` b
     2.  Perform a final sanity check that everything looks ok.
 
         ```
-        cargo check --all-targets
+        cargo hack --exclude-all-features --each-feature check --all-targets --workspace
```
Comment (niklasad1) [inline RELEASING.md:62]: doesn't work anymore with `wasm stuff`

File: RELEASING.md [hunk 6]
```diff
@@ -13,15 +13,15 @@ We also assume that ongoing work done is being merged directly to the `master` b
...
-    cargo doc --open --all-features
+    cargo doc --open
     ```
 
     CI checks for broken internal links at the moment. Optionally you can also confirm that any external links
     are still valid like so:
 
     ```
     cargo install cargo-deadlinks
-    cargo deadlinks --check-http -- --all-features
```
Comment (niklasad1) [inline RELEASING.md:24]: doesn't work anymore with `wasm stuff`

File: RELEASING.md [hunk 7]
```diff
@@ -13,15 +13,15 @@ We also assume that ongoing work done is being merged directly to the `master` b
 3.  Check that you're happy with the current documentation.
 
     ```
-    cargo doc --open --all-features
+    cargo doc --open
```
Comment (niklasad1) [inline RELEASING.md:16]: doesn't work anymore with `wasm stuff`

**Review by SkymanOne — APPROVED** [review]

**Pull request merged**